pkg/rpc/client/native.go

```diff
@@ -3,9 +3,14 @@ package client
 // Various non-policy things from native contracts.
 
 import (
+	"crypto/elliptic"
 	"fmt"
 
+	"github.com/nspcc-dev/neo-go/pkg/core/native"
```

Comment:
Not a good dependency for `rpc/client`, although I see that it's already there for `native.NotaryVerificationPrice`.

Reply:
We may duplicate these constants inside the RPC client, but I'm not sure it's a good idea.

Reply:
Constants can always be moved into package of their own.